src/System.Diagnostics.PerformanceCounter/ref/System.Diagnostics.PerformanceCounter.csproj

```diff
@@ -4,6 +4,9 @@
     <ProjectGuid>{5BDC8641-E3EE-47B5-BE7B-2D2275402412}</ProjectGuid>
     <Configurations>net461-Debug;net461-Release;netfx-Debug;netfx-Release;netstandard-Debug;netstandard-Release</Configurations>
   </PropertyGroup>
+  <PropertyGroup>
```

Comment:
Instead of adding a new `PropertyGroup` just add this property in the group above.

Reply:
fixed